test/e2e/resources/fleet-capi-test.yaml

```diff
@@ -0,0 +1,4896 @@
+---
```

Comment:
how is it related to fleet?

Reply:
https://github.com/richardcase/fleet-clusters